### PR TITLE
RollLogin: use in BottomSheetModal

### PR DESCRIPTION
**Pull request by MattNguyen**
Rather than displaying RollLogin when the user navigates to a post, we
now preemptively display the RollLogin component in a BottomSheetModal
before the user is routed by checking their isAuthenticated state.

- Add wallet icon to Feed page nav when the user hasn't authenticated
  with Roll. We will probably need to make this persistent as this is
  what the designs call for.
- Create BottomSheetModal. This still needs work to support gestures but
  the user can still tap out of the modal.
- Ensure user is redirected to the post they clicked on after roll auth

Screen Recording 2020-08-23 at 11 35 PM

### CODE REVIEW

File: components/Page.js [hunk 1]
```diff
@@ -36,6 +40,9 @@ function Page() {
       console.log(e);
     }
   }, []);
+  useEffect(() => {
+    if (ostate.user.balances.length === 0) actions.refreshUser();
```
Comment (MattNguyen) [inline components/Page.js:44]: Is this the best way to check if the user is logged in or not?

File: tailwind.config.js [hunk 2]
```diff
@@ -10,17 +10,20 @@ module.exports = {
     ],
   },
   theme: {
+    borderRadius: {
+      xl: "2rem",
```
Comment (MattNguyen) [inline tailwind.config.js:14]: @jennifertrin this isn't working. any idea how to make it so that i can use `rounded-t-xl`?

Comment (jennifertrin) [reply tailwind.config.js:14]: Let me check! I will not be back at my computer until at least another two hours.

File: components/Page.js (hunk 1 above)
Comment (jadbox) [reply components/Page.js:44]: It's not ideal, I'll make a note to add an explicit flag for it later :)

**Comment by jadbox** [comment]
LGTM to merge in, we can always tweak it a little later

File: tailwind.config.js (hunk 2 above)
Comment (jennifertrin) [reply tailwind.config.js:14]: Hello @MattNguyen,

Can you please try "2 rem" - looks like you are missing a space between the "2" and "rem"

 ```
theme: {
    borderRadius: {
      xl: "2 rem",
```

Please let me know if that works!

Comment (jennifertrin) [reply tailwind.config.js:14]: Wait, @MattNguyen, nevermind on that previous comment! I'll do some troubleshooting on my end.

Comment (jennifertrin) [reply tailwind.config.js:14]: Hello @MattNguyen,

The correct syntax should be:

```
borderRadius: {
      'xl': '2rem',
    }
```

However, it is still not working on my end. Can you please try this and let me know if it works?

Thank you!

**Pull request merged**